plugins/Amplifier/AmplifierControlDialog.h

```diff
@@ -43,4 +48,9 @@ class AmplifierControlDialog : public EffectControlDialog
 
 } ;
 
+
+} // namespace gui
+
+} // namespace plugin
```

Comment:
It's namespace `lmms` (but maybe it was planned to go for a `plugin` namespace?)

Reply:
As @JohannesLorenz mentioned, this PR only implements two namespaces: `lmms` and the nested `lmms::gui`. I believe a `plugin` namespace was considered at one point, this namespace-end comment was probably a remnant of that. @allejok96 took care of it and corrected it to be `lmms` as intended.